servant-auth-server/servant-auth-server.cabal

```diff
@@ -1,5 +1,5 @@
 name:           servant-auth-server
-version:        0.4.0.0
+version:        0.4.0.1
```

Comment:
Could you please leave this change out? we're now trying to be consistent over all the servant packages about not updating versions or changelogs in PRs, this is all taken care of at release time.